htdocs/core/lib/date.lib.php

```diff
@@ -137,11 +137,11 @@ function dol_time_plus_duree($time, $duration_value, $duration_unit)
 	$deltastring = 'P';
 
 	if ($duration_value > 0) {
-		$deltastring .= abs($duration_value);
+		$deltastring .= abs((int) $duration_value);
```

Comment:
duration_value and duration_unit should be already int according to method doc.
If not, it would be better to fix the caller. Here the fix may "hide" a problem into caller so i prefer to keep current code as it is to alert when a call is not done with correct parameters.